Makefile

```diff
@@ -283,14 +285,18 @@ web: web-pre-process $(HUGO)
 	# TODO(bwplotka): Make it --gc
 	@cd $(WEB_DIR) && HUGO_ENV=production $(HUGO) --config hugo.yaml --minify -v -b $(WEBSITE_BASE_URL)
 
+.PHONY: failint
+failint:
+	@echo ">> verifying packages being imported"
+	@faillint -paths $(IMPORTS_TO_AVOID) ./...
```

Comment:
What about vendor? I think we should exclude vendor

Reply:
If we don't want to run it across all folders, we would have to specify the files and directories manually (or a for loop): pkg, cmd, test, scripts and doc.go. Other option could be, opening a PR to the faillint project with this idea.

What do you think?

Reply:
ah.. TBH can we do that? Can we ask question on upstream project how to avoid vendor dir?

Reply:
Maybe it already support this?

Reply:
Current version has no option like that. I'm going to ask in the upstream project to see what they recommend.

Reply:
According to the project developer, faillint excludes the `vendor` directory automatically.

Reply:
Thanks for checking :heart: